.github/workflows/nightly-ci.yml

```
@@ -62,6 +62,7 @@ jobs:
         run: |
           docker build -f package/Dockerfile -t rancher/fleet:dev --build-arg="ARCH=$GOARCH" .
           docker build -f package/Dockerfile.agent -t rancher/fleet-agent:dev --build-arg="ARCH=$GOARCH" .
+          cd e2e/assets/gitrepo
           DOCKER_BUILDKIT=1 docker build -f Dockerfile.gitserver -t nginx-git:test --build-arg="passwd=$(openssl passwd foo)" .
```

```suggestion
          DOCKER_BUILDKIT=1 docker build -f e2e/assets/gitrepo/Dockerfile.gitserver -t nginx-git:test --build-arg="passwd=$(openssl passwd foo)" .
```

wouldn't it be better to specify the path in the -f paramenter as we do for `package/Dockerfile` and `package/Dockerfile.agent`?

---

We could do that, provided that we also specify the right context for `nginx_git.conf` to be resolved and copied into the image, ie:
```suggestion
          DOCKER_BUILDKIT=1 docker build -f e2e/assets/gitrepo/Dockerfile.gitserver -t nginx-git:test --build-arg="passwd=$(openssl passwd foo)" e2e/assets/gitrepo
```

Happy to make that change if you still think it makes more sense than using `cd` (I initially chose this approach for conciseness).

---

that looks good!

---

Done :)

---

Thanks!